src/main/java/com/minelittlepony/model/player/PlayerModels.java

```diff
@@ -0,0 +1,41 @@
+package com.minelittlepony.model.player;
+
+import com.minelittlepony.model.PMAPI;
+import com.minelittlepony.model.ModelWrapper;
+
+public enum PlayerModels {
+    /**
+     * @deprecated Will be removed in a later revision
+     */
+    @Deprecated HUMAN("default", "slim", () -> PMAPI.pony, () -> PMAPI.ponySmall),
+    EARTH("earthpony", "slimearthpony", () -> PMAPI.earthpony, () -> PMAPI.earthponySmall),
```

Comment:
"slimearthpony"

They wouldn't be so slimy if they stopped rolling around in the mud.

Reply:
Lol. I saw that too. Of course I kept it.